mods/default/functions.lua

```diff
@@ -724,9 +724,9 @@ local log_non_player_actions = minetest.settings:get_bool("log_non_player_action
 -- `vector.check` is available since minetest >= 5.5.0
 local is_vector = vector.check or function(v)
 	return v and type(v) == "table" and
-		v.x or type(v.x) == "number" and
-		v.y or type(v.y) == "number" and
-		v.z or type(v.z) == "number"
+		v.x and type(v.x) == "number" and
```

Comment:
You don't need to check for `v.x`; if `v.x` is `nil`, `type(v.x)` will be `"nil" ~= "number"`.

Reply:
Thanks, fixed in https://github.com/minetest/minetest_game/pull/2941/commits/76c67ac84eeedebcdb3b7dcc02c8965faaf21d65